sample_apps/run_example.sh

```diff
@@ -426,10 +778,13 @@ function build_simple_server() {
 
     # Compile the oelib for OE host verification
     # Likely, user does not have OE SDK installed or does not want to enable OE:
-    # This should produce a go file for the certifier protobufs
+    # This should produce a Go file for the certifier protobufs
     # called certifier.pb.go in certprotos.
+    make_arg=""
+    if [ "${SampleAppName}" = "simple_app" ]; then make_arg=dummy; fi
```

Comment:
We will do real `make` for OE-simple-app but will do `make dummy` for `simple_app`.